package.json

```diff
@@ -26,8 +26,9 @@
   },
   "homepage": "https://github.com/exoplay/exobot-adapter-discord#readme",
   "dependencies": {
-    "discord.js": "^9.3.0",
-    "@exoplay/exobot": ">=4.1 <5"
+    "@exoplay/exobot": ">=4.1 <5",
+    "discord.js": "^11.0.0",
```

Comment:
I was worried about the auto-reconnect breaking with uws.  When they put out the first patch for v11 I will revisit this.